### PR TITLE
Add a model management notebook

### PR DESCRIPTION
**Pull request by igor-davidyuk**
This PR introduces a new example notebook showcasing work with models and algorithms

### CODE REVIEW

File: notebooks/013_manage_models.ipynb [hunk 1]
```diff
@@ -0,0 +1,403 @@
...
+    "Remember we performed an optimization job for the model? Now we can also retrieve the optimized model from the model object."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "optimized_model = model.get_optimized_model(optimization_type=\"pot\")\n",
```
Comment (ljcornel) [inline notebooks/013_manage_models.ipynb:295]: In addition, there is the `ModelClient.get_latest_optimized_model()` method that allows filtering by algo name, optimization_type, precision and xai capabilities.
Might be good to show that one as well, as an alternative approach.

**Review by ljcornel — COMMENTED** [review]
Looks good to me, thanks for adding this! I've added a few minor comments, but I think the overall approach is pretty good. 

I realize it may be sensitive to changes in the default algorithms, or certain user changes in the project it relies on. But I don't think it's a big risk, it should still run but maybe lose some of it's effectiveness in certain cases.

File: notebooks/013_manage_models.ipynb [hunk 2]
```diff
@@ -0,0 +1,403 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# GETi Model Management\n",
```
[the remark this answers is not in the document]
Comment (igor-davidyuk) [reply notebooks/013_manage_models.ipynb:7]: Got it!

File: notebooks/013_manage_models.ipynb [hunk 3]
```diff
@@ -0,0 +1,403 @@
...
+    "We can also ask the server to return a model in the reduced precision format, which can be used for deployment on some edge devices."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "model.get_optimized_model(precision=\"FP16\")"
```
[the remark this answers is not in the document]
Comment (igor-davidyuk) [reply notebooks/013_manage_models.ipynb:312]: Added explanations for it

File: notebooks/013_manage_models.ipynb [hunk 4]
```diff
@@ -0,0 +1,403 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "model_summary = model_groups[1].models[0]\n",
+    "model = model_client.update_model_detail(model_summary)"
```
[the remark this answers is not in the document]
Comment (igor-davidyuk) [reply notebooks/013_manage_models.ipynb:279]: added

**Review by ljcornel — COMMENTED** [review]
Looks good to me, could you just update the spelling of Geti (GETi -> Geti)?

**Comment by igor-davidyuk** [comment]
> Looks good to me, could you just update the spelling of Geti (GETi -> Geti)?

Done!

File: notebooks/013_manage_models.ipynb (hunk 1 above)
Comment (igor-davidyuk) [reply notebooks/013_manage_models.ipynb]: done!

**Review by ljcornel — APPROVED** [review]

**Pull request merged**